test/development/acceptance-app/ReactRefreshLogBoxMisc.test.ts

```diff
@@ -3,210 +3,15 @@ import { FileRef, nextTestSetup } from 'e2e-utils'
 import path from 'path'
 import { outdent } from 'outdent'
 
-// TODO: re-enable these tests after figuring out what is causing
-// them to be so unreliable in CI
-describe.skip('ReactRefreshLogBox app', () => {
-  const { next } = nextTestSetup({
+describe('ReactRefreshLogBox app', () => {
+  const { isTurbopack, next } = nextTestSetup({
     files: new FileRef(path.join(__dirname, 'fixtures', 'default-template')),
     skipStart: true,
   })
 
-  test('<Link> with multiple children', async () => {
```

Comment:
Tested in https://github.com/vercel/next.js/blob/aae3692ce7b473716690a71cbd54c7d7e3552abd/test/e2e/next-link-errors/next-link-errors.test.ts#L64